azurerm/internal/services/kusto/tests/kusto_table_resource_test.go

```diff
@@ -0,0 +1,201 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/azure-kusto-go/kusto"
+	"github.com/Azure/azure-kusto-go/kusto/data/table"
+	"github.com/Azure/azure-kusto-go/kusto/unsafe"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/kusto/parse"
+	dataplaneTypes "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/kusto/types"
+)
+
+func TestAccAzureRMKustoTable_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_kusto_table", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMKustoTableDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMKustoTable_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMKustoTableExists(data.ResourceName),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
```

Comment:
in general, we will have basic, complete and update test, could you please add more tests

Reply:
done